tests/aws/services/lambda_/test_lambda_common.py

```diff
@@ -246,30 +268,57 @@ def test_runtime_wrapper_invoke(self, multiruntime_lambda, snapshot, tmp_path, a
         assert invocation_result_payload["environment"]["WRAPPER_VAR"] == test_value
 
 
-# TODO: Split this and move to PRO
-@pytest.mark.skip(reason="this test is broken and is being worked on")
+# TODO: remove this once all non-arm compatible runtimes are deprecated by the end of 2023
+x86_runtimes = [
```

Comment:
Can we rename those? It was quite unclear for me that these are only relevant for the test_manual_endpoint_injection test.